### ISSUE TITLE
Get all blooks in the game for ever (Unlock all the blooks)

### ISSUE THREAD

**Issue by AdrienHacker** [rxzyx/Blooket-Hacks#108]
Get all blooks in the game for ever (Unlock all the blooks)

Thanks:)

**Comment by rxzyx** [comment]
Impossible unless you can save the value somewhere.

**Comment by rxzyx** [comment]
I can try this later on.

**Comment by rxzyx** [comment]
Just for you to know this will only be client-sided. Not server-sided.

**Comment by rxzyx** [comment]
Also how is this an issue?

**Comment by Wilke000** [comment]
It’s not lol

**Comment by AdrienHacker** [comment]
Sorry for putting it in issus

**Comment by Wilke000** [comment]
Where else are you supposed to put it there is no discussions *pls add that btw rxzyx that would help people like me who don’t have discord*

**Comment by rxzyx** [comment]
Discussions? ROFL. Sorry, not paying for my own private server lol. Make a discord account if you want to make a suggestion, you made a github one, how is it so hard?

**Comment by AdrienHacker** [comment]
This issu/request will help everyone

**Comment by AdrienHacker** [comment]
Think about it pls

**Comment by rxzyx** [comment]
I can do it but there is no reason to.

**Comment by AdrienHacker** [comment]
Why?

**Comment by rxzyx** [comment]
Plus, “client-sided”

**Comment by rxzyx** [comment]
> Why?

“no reason to”, not good enough?

**Comment by AdrienHacker** [comment]
What does that mean

**Comment by rxzyx** [comment]
To dumb it down something stored on the server

**Comment by AdrienHacker** [comment]
client sided

**Comment by AdrienHacker** [comment]
I searched for 3 months looking for someone who makes hacks that work.

**Comment by AdrienHacker** [comment]
Hi?

**Comment by rxzyx** [comment]
And you found it after like idk 2 min, so ok simple answer no now don’t comment back.

**Comment by Wilke000** [comment]
@AdrienHacker 
[Use these](https://github.com/Minesraft2/Blooket-Cheats/blob/main/obfuscated/gui.js)

**Comment by rxzyx** [comment]
Promotions are not allowed.

**Comment by Wilke000** [comment]
> Promotions are not allowed.

Yea i knew u were gonna do that

**Comment by AdrienHacker** [comment]
![Screenshot (13)](https://user-images.githubusercontent.com/115197019/197587830-68cc2d20-db11-44ba-ae8e-af8cb182f871.png)

**Comment by AdrienHacker** [comment]
I just started after the update and that's all I have
